### ISSUE TITLE
Missing token kinds for argument specifiers

### ISSUE THREAD

**Issue by kyouko-taiga** [colibri-lang/Colibri#3]
Swift has 2 private argument specifiers, namely `__shared` and `__owned`, for which there isn't any token kind yet.

The semantics of these specifiers is defined in the [Ownership Manifesto](https://github.com/apple/swift/blob/master/docs/OwnershipManifesto.md#shared-values).

**Comment by coetaur0** [comment]
The latest version of the lexer in branch [refactoring/lexer](https://github.com/colibri-lang/Colibri/blob/refactoring/lexer/Sources/Parser/TokenKind.swift) solves this issue.

**Comment by kyouko-taiga** [comment]
There seems to be a problem with the tests in that branch. Some cases of `TokenKind` are missing.

**Comment by coetaur0** [comment]
Yes, I’ll have to update the tests to match the many changes made to the lexer.
